src/rpc/format.rs

```diff
@@ -75,9 +88,13 @@ impl HttpFormat for Format {
 		if matches!(self, Format::Json) {
 			// If this has significant performance overhead it could be replaced with unsafe { String::from_utf8_unchecked(res) }
 			// This would be safe as in the case of JSON res come from a call to Into::<Vec<u8>> for String
-			Ok((StatusCode::OK, String::from_utf8(res).unwrap()).into_response())
+			Ok((
+				[(CONTENT_TYPE, HeaderValue::from(Accept::ApplicationJson))],
```

Comment:
```suggestion
				[(CONTENT_TYPE, HeaderValue::from(ContentType::ApplicationJson))],
```
I think it makes more sense to use ContentType for this instead of Accept

Reply:
Indeed. I just c/c-ed code from `output.rs`.